extensions/dicom-microscopy/src/DicomMicroscopyANNSopClassHandler.js

```diff
@@ -0,0 +1,103 @@
+import OHIF, { DicomMetadataStore } from '@ohif/core';
+import loadAnnotation from './utils/loadAnnotation';
+import getSourceDisplaySet from './utils/getSourceDisplaySet';
+
+const { utils } = OHIF;
+
+const SOP_CLASS_UIDS = {
+  MICROSCOPY_BULK_SIMPLE_ANNOTATION: '1.2.840.10008.5.1.4.1.1.91.1',
+};
+
+const SOPClassHandlerId =
+  '@ohif/extension-dicom-microscopy.sopClassHandlerModule.DicomMicroscopyANNSopClassHandler';
+
+function _getDisplaySetsFromSeries(instances, servicesManager, extensionManager) {
+  // If the series has no instances, stop here
+  if (!instances || !instances.length) {
+    throw new Error('No instances were provided');
+  }
+
+  const { displaySetService, microscopyService } = servicesManager.services;
+
+  const instance = instances[0];
+
+  const naturalizedDataset = DicomMetadataStore.getSeries(
+    instance.StudyInstanceUID,
+    instance.SeriesInstanceUID
+  ).instances[0];
+
+  const {
+    SeriesDescription,
+    ContentDate,
+    ContentTime,
+    SeriesNumber,
+    StudyInstanceUID,
+    SeriesInstanceUID,
+    SOPInstanceUID,
+    SOPClassUID,
+  } = instance;
+
+  const displaySet = {
```

Comment:
Add thumbnailSrc: null  and if Alireza has merged his overlay stuff, then add the overlay headers.

Reply:
It means something specific - null in that case means this object can't render a thumbnail so even if it gets included in the study browser, it won't try to render the thumbnail.

Reply:
Updated.